drivers/virtualbox/virtualbox.go

```diff
@@ -553,9 +551,8 @@ func (d *Driver) diskPath() string {
 // Make a boot2docker VM disk image.
 func (d *Driver) generateDiskImage(size int) error {
 	log.Debugf("Creating %d MB hard disk image...", size)
-
-	magicString := "boot2docker, please format-me"
-
+	parts := strings.Split(isoFilename, ".")
+	magicString := fmt.Sprintf("%s, please format-me", parts[0])
```

Comment:
@sthulb thanks for your review.I have removed code for dynamically set magic string.